src/main/resources/org/jenkinsci/plugins/workflow/cps/Snippetizer/workflow.css

```diff
@@ -99,6 +99,10 @@ z.root>dt {
   padding-bottom: 0
 }
 
+.snippetizer-help {
```

Comment:
this css file appears not to be loaded on http://localhost:8083/jenkins/job/pipeline/pipeline-syntax/

the css wasn't applied, and when I searched the dom I couldn't find the css file
![image](https://user-images.githubusercontent.com/21194782/107928324-11c3d880-6f70-11eb-93f9-2574377e8f42.png)

Reply:
@timja Okay, so I still don't have that right.

Reply:
@timja 
I agree it doesn't look great, but I feel this should not block merging the PR.  Would you accept that or do we need to address it in this PR?  Either way, I'm going to bed for the night, back online in about six hours.

Reply:
I have a proposal on https://github.com/jenkinsci/workflow-cps-plugin/pull/411#issuecomment-779145393 . I used inline styling because I didn't want to get into why the css wasn't loaded. if the file is not being used we might as well remove it.

Reply:
Based on some examples in Jenkins core, I think to include the CSS you'd need to add:

```xml
<st:adjunct includes="org.jenkinsci.plugins.workflow.cps.Snippetizer.workflow"/>
```